exercises/scrabble-score/test/scrabble_score_test.dart

```diff
@@ -0,0 +1,168 @@
+import 'package:test/test.dart';
+import 'package:scrabble_score/scrabble_score.dart';
+
+void main() {
+  group('Scrabble score', () {
+    group('should return a score of 0 for', () {
+      test('an empty string', () {
+        var word = new ScrabbleScore('');
+        expect(word.getScrabbleScore(), equals(0));
+      });
+
+      test('strings containing only whitespace characters', () {
+        var word = new ScrabbleScore(' \t\r\n');
+        expect(word.getScrabbleScore(), equals(0));
+      });
```

Comment:
Add `skip: true` to all tests after the first.

Add `skip: false` to the first test.